src/Compilers/Test/Core/TargetFrameworkUtil.cs

```diff
@@ -171,9 +171,11 @@ static TargetFrameworkUtil()
             TargetFramework.Empty => ImmutableArray<MetadataReference>.Empty,
             TargetFramework.NetStandard20 => NetStandard20References,
             TargetFramework.Net50 => ImmutableArray.CreateRange<MetadataReference>(LoadDynamicReferences("Net50")),
-            TargetFramework.Net60 => ImmutableArray.CreateRange<MetadataReference>(Net60.All),
+            TargetFramework.Net60 => ImmutableArray.CreateRange<MetadataReference>(LoadDynamicReferences("Net60")),
```

Comment:
Should we use dynamic loading for Net70 instead of for Net60, since the IDE tests presumably use Net60?
Alternatively, should we go ahead and dynamically load Net70 too?

Reply:
The IDE tests actually use `net70`, not `net60`. The IDE tests are written against how user code will behave. Essentially what experience do we get when a user targets say `net7.0`. That is what these assemblies represent. They don't represent the code that runtime is necessarily executing on.

Reply:
As for whether or not to dynamically load `net70`, the expectation is that our test code should be able to easily access the latest .NET Framework and .NET Core TFM. As such we load them in the base utility library. Anything outside those should be dynamically loaded.